library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -124,6 +124,34 @@ def store
       store_autostart
     end
 
+    # Returns the service widget help text
+    #
+    # @return [String]
+    def help
+      helptext = "<h2>Service configuration</h2>"
+      # TRANSLATORS: helptext for the service current status
+      helptext << _(
+        "<h3>Current status</h3>" \
+        "Displays the curren status of the service."
+      )
+
+      # TRANSLATORS: helptext for the "After writting configuration" service widget option
+      helptext << _(
+        "<h3>After writting configuration</h3>" \
+        "Allow to change the service status immediately after accepting the changes. Available
+        options depend on the current state. The <b>Keep current state</b> special action leaves the
+        service state untouched."
+      )
+
+      # TRANSLATORS: helptext for the "After reboot" service widget option
+      helptext << _(
+        "<h3>After reboot</h3>" \
+        "Let choose if service should be started automatically on boot. Some services could be
+        configured <b>on demand</b>, which means that the associated socket will be running and in
+        charge of start the service if needed."
```

Comment:
maybe I would simplify this with `will be running and start the service if needed.`

Reply:
Done!